backend/src/tracking/manager.py

```diff
@@ -0,0 +1,36 @@
+"""Handles the camera processing."""
+
+from threading import Thread
+from .camera import Camera
+
+
+class TrackingManager:
+    """The tracking manager can start or stop the camera tracking and forward callbacks."""
+
+    def __init__(self):
+        self.camera = None
+        self.thread = None
+
+    def start_tracking(self) -> None:
+        """Start the camera tracking."""
+        self.camera = Camera(1)
```

Comment:
Is the `id` always 1 for the camera module?

Reply:
That's actually only on my computer for testing purposes, it may vary on other systems.
But when we use the `picamera` module, we don't have to mess with camera ids anymore since the PI only has one CSI. So I just let it here hard-coded as we will remove it when we have the actual hardware.

Reply:
Ahh I see that makes sense, thanks for the clarification 👍